msteams-platform/toolkit/debug-local.md

```diff
@@ -18,25 +18,25 @@ Teams Toolkit helps you to debug and preview your Teams app locally. Debug is th
 * [Stop debugging](#stop-debugging)  
 
 
-During the debug process, Teams Toolkit automatically starts app services, launches debuggers, and sideloads the Teams app. The Teams app is available for preview in Teams web client locally after debugging. You can also customize debug settings to use your bot endpoints, development certificate, or debug partial component to load your configured app.
+During the debug process, Teams Toolkit automatically starts app services, launches debuggers, and sideloads the Teams app. You can also customize the debug settings to use your bot endpoints, development certificate, or debug partial component to load your configured app. The Teams app is available for preview in Teams web client locally after the debugging.
 
 ## Prerequisite
 
 Install the [latest version of Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).
 
-## Key features of Teams Toolkit
-
+## Key debug features
+The following list shows the primary debug features used in Teams Toolkit.
```

Comment:
There is no need to add the sentence as the heading provides the required information.